pyanaconda/bootloader.py

```diff
@@ -1682,7 +1682,6 @@ def check(self):
 class EFIBase(object):
 
     def __init__(self):
-        super(EFIBase, self).__init__()
```

Comment:
Can do, I'm a bit confused though.  since it inherits from `object` my understanding is that `__init__` for that class was just `pass`

Reply:
Yes, it is confusing. The difference is when you use this class as one of the parents of another class, for example `EFIGRUB1`.

Try this code:

```
class X(object):
  def __init__(self):
    super().__init__()
    print("X")

class Y(object):
  def __init__(self):
    super().__init__()
    print("Y")

class Z(Y, X):
  def __init__(self):
    super().__init__()
    print("Z")

Z()
```

It will print `X Y Z`.

Reply:
Neat! I was unaware